### PR TITLE
[ASAA-131] Component Weather Widget (Daily & Hourly Forecast)

### PR DESCRIPTION
**Pull request by MMurrayAtBR**
### [[ASAA-131]](https://bottlerocketstudios.atlassian.net/browse/ASAA-131) Component Weather Widget (Daily & Hourly Forecast)

- Added `Light Blue` color to Colors.kt. Needed for precipitation probability
- Added `grid_2_25` and `grid_3_75` to `Dimens` file.
- Added a content description for weather icons to strings.xml
- Created ForecastChip.kt as the main wrapper to house the Daily and Hourly forecast data
- Created `DailyForecastChip` and `HourlyForecastChip` to display the forecast data. Text sizing differences require different layouts
- Created state files for both `DailyForecastChip` and `HourlyForecastChip`
- Created Preview files for both forecast chips
- Added weather icons

### CODE REVIEW

**Review by BottleRocket-Colin — APPROVED** [review]

File: shared/src/commonMain/kotlin/com/br/kmmdemo/forecasts/WeatherEnum.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.br.kmmdemo.forecasts
+
+import com.br.kmmdemo.resources.SharedRes
+import dev.icerock.moko.resources.ImageResource
+
```
Comment (VanyaHuamanBottleRocket) [inline shared/src/commonMain/kotlin/com/br/kmmdemo/forecasts/WeatherEnum.kt:5]: Should these weather text be moved to the strings resources

Comment (BottleRocket-Colin) [reply shared/src/commonMain/kotlin/com/br/kmmdemo/forecasts/WeatherEnum.kt:5]: Good catch. @MMurrayAtBR Could you do a quick change to get the strings moved into resources?

**Review by BottleRocket-Colin — CHANGES_REQUESTED** [review]
Minor resource change

File: shared/src/commonMain/kotlin/com/br/kmmdemo/forecasts/WeatherEnum.kt (hunk 1 above)
Comment (MMurrayAtBR) [reply shared/src/commonMain/kotlin/com/br/kmmdemo/forecasts/WeatherEnum.kt:5]: stringResource is a composable function that can not be called from a standard Enum. I can move them to constants, though

**Review by BottleRocket-Colin — APPROVED** [review]

**Pull request merged**